app/models/user.rb

```diff
@@ -2,7 +2,7 @@ class User < ActiveRecord::Base
   has_many :jobs
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable,
-         :omniauthable
+         :omniauthable, :omniauth_providers => [:linkedin]
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.